src/helpers/get-batch-page-keys.ts

```diff
@@ -0,0 +1,25 @@
+export const getBatchPageKeys = (numBatches: number, pagesPerBatch: number, pageSize: number, limit?: number | undefined): any[] => {
+  const pageKeys = [];
+  for (let i = 0; i < numBatches; i++) {
+    pageKeys.push({
+      limit: getPageSize(pageSize, i, numBatches, limit, pagesPerBatch),
+        // Start at 1
+        startindex: 1 + (i * pagesPerBatch * pageSize),
```

Comment:
I think there was a tabbing issue from the previous code... would you mind shifting lines 6-7 back 1 tab?

Reply:
Fixed